ecl/dedicated_hypervisor/v1/_proxy.py

```diff
@@ -264,3 +265,62 @@ def update_cfgw_connection(self, server_id):
         """
         cfgw = _server.CFGWConnection()
         return cfgw.update_cfgw_connection(self.session, server_id)
+
+    def show_vcenter(self):
+        """
+        Shows the registered vCenter server information.
+
+        :return: One :class:`~ecl.dedicated_hypervisor.v1.vcenter.show_vcenter`
+        instance.
```

Comment:
```suggestion
        :return: :class:`~ecl.dedicated_hypervisor.v1.vcenter.VCenter`
```
- Show Registered vCenter Information API はリストを返すので `One` ではないです。
- `:class:` の後にはクラスパスを記載してください。

Reply:
returnのクラスパスの記載を修正しました。